src/components/PlioQuestion.vue

```diff
@@ -184,6 +187,12 @@ export default {
         return this.progressBarInfo['config']['enabled']
       
       return false;
+    },
+    userAnswerIndex() {
+      // index of user answer in the options array
+      return this.plioQuestion.item.question.options.indexOf(
+        this.plioQuestion.user_answer
```

Comment:
Returns the index of the user answer in the options array. Why this is needed now?
Earlier, when coming back to an already answered question, there was a need to select the option which was the user's answer. And that was being done by a selector which selected the element based on the option text.

But now, as the option text can contain random latex character and random strings, it's best to select those options using their index.